zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/RestoreFromBackupTool.java

```diff
@@ -45,6 +51,10 @@
 public class RestoreFromBackupTool {
   private static final Logger LOG = LoggerFactory.getLogger(RestoreFromBackupTool.class);
 
+  private static final int MAX_RETRIES = 10;
+  private static final String LATEST = "latest";
```

Comment:
I saw this key words for another class as well. Can we have a common place for that? Otherwise, if we changed the name, we may forget to change somewhere.

Reply:
Done. Moved this string along with TimetableUtil.LATEST to BackupUtil.LATEST.